KSP-Backend/KSP/mvp/models.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+from django.db import models
+
+YEAR_CHOICES = []
+for year in range(2016, (datetime.datetime.now().year + 1)):
+    YEAR_CHOICES.append((year, year))
+
+
+class Notes(models.Model):

```

Comment:
Notes does not have field for storing upvotes/downvotes

Reply:
I will be adding these.